generator/golang/scope_internal.go

```diff
@@ -497,12 +510,32 @@ func (s *Scope) resolveTypesAndValues(cu *CodeUtils) {
 }
 
 func isNestedField(f *parser.Field) bool {
-	annos := f.Annotations.Get(nestedAnnotation)
-	if len(annos) == 0 {
+	return isBoolAnnotationSet(f.Annotations, nestedAnnotation, true)
+}
+
+func isInterfaceField(f *parser.Field) bool {
+	return isBoolAnnotationSet(f.Annotations, interfaceAnnotation, true)
+}
+
+func isAliasType(f *parser.StructLike) bool {
+	return isBoolAnnotationSet(f.Annotations, aliasAnnotation, true)
+}
+
+func isBoolAnnotationSet(annos parser.Annotations, anno string, flag bool) bool {
```

Comment:
Both the name and impl are not easy to understand.

I suggest `annotationContainsTrue` and displays a warning (or even panic) if the annotation set contains multiple values.

Reply:
done.